python/strings_udf/cpp/CMakeLists.txt

```diff
@@ -0,0 +1,108 @@
+# =============================================================================
```

Comment:
I'm questioning if this package should be structured like `libcudf_kafka` which keeps all its C++ files in `cudf/cpp/libcudf_kafka`. It would be nice if we could avoid having the directory `python/strings_udf/cpp`.

Reply:
This has been discussed many times. I don't know if I would be able to summarize all the discussions and decisions made to get to this point. Perhaps re-opening the discussion again may be outside the scope of PR comments.

Reply:
Here is a link to the meeting where this was decided.
https://nvidia-my.sharepoint.com/:v:/p/bdice/EX3vMzEmYOVIsKJZLrCLl_sBS4Wb6OtG5bSi_2Soi3mIjg
Let me know if you have any issues with getting access to the recording and I can ask the creator of the recording to grant you access. :-)

Reply:
No problem. I'll resolve this and it can be revisited in the future if / when needed. 👍